.github/workflows/tests.yml

```diff
@@ -18,7 +18,8 @@ jobs:
       - name: Install PHP
         uses: shivammathur/setup-php@v2
         with:
-          php-version: 8.0
+          php-version: 8.1
+          coverage: none
```

Comment:
By default when non setting `coverage: none` xdebug is installed, so this just avoids